teku/src/main/java/tech/pegasys/teku/cli/options/LoggingOptions.java

```diff
@@ -67,7 +65,7 @@
       paramLabel = "<FILENAME>",
       description = "Path containing the location (relative or absolute) and the log filename.",
```

Comment:
Need to update the description to specify what the default will be (e.g. `<data-path>/logs/teku.logs`) as PicoCLI won't be able to display it for us anymore.  Also add `showDefaultValue = Visibility.NEVER` to the annotation to prevent PicoCLI reporting the default as null.

Reply:
Done.